sanic/response.py

```diff
@@ -97,6 +97,8 @@ def output(self, version="1.1", keep_alive=False, keep_alive_timeout=None):
         headers = b''
         if self.headers:
             headers = b''.join(
+                b'%b: %b\r\n' % (name.encode(), str(value).encode('utf-8'))
+                if type(value) != Cookie else
```

Comment:
May I ask you to avoid use `type` here? I think `isinstance` should work (but please check this I could be wrong). And one more thing: the only difference here is in `str(value)`, so could you please just create additional variable and move `if ... else..` block there to avoid code duplication?